app/assets/javascripts/pages/errors.js

```diff
@@ -0,0 +1,88 @@
+var ready = function () {
+//initialize and create the chart
+  AmCharts.makeChart("chartdiv", {
+    "type": "serial",
+    "theme": "light",
+    "marginRight": 40,
+    "marginLeft": 40,
+    "autoMarginOffset": 20,
+    "dataDateFormat": "YYYY-MM-DD",
+    "color": "#61656b",
+    "fontSize": 12,
+    "fontFamily": "Roboto",
+    "valueAxes": [{
+        "id": "v1",
+        "axisAlpha": 0,
+        "position": "left",
+        "ignoreAxisWidth":true
+    }],
+    "balloon": {
+        "borderThickness": 1,
+        "shadowAlpha": 0
+    },
+    "graphs": [{
+        "id": "g1",
+        "balloon":{
+          "drop":true,
+          "adjustBorderColor":false,
+          "color": "#ffffff"
+        },
+        "bullet": "round",
+        "bulletBorderAlpha": 1,
+        "bulletColor": "#FFFFFF",
+        "bulletSize": 5,
+        "hideBulletsCount": 50,
+        "lineColor": "#3ba3f3",
+        "lineThickness": 3,
+        "title": "red line",
+        "useLineColorForBulletBorder": true,
+        "valueField": "value",
+        "balloonText": "<span style='font-size:18px;'>[[value]]</span>"
+    }],
+    "chartCursor": {
+        "pan": true,
+        "valueLineEnabled": true,
+        "valueLineBalloonEnabled": true,
+        "cursorAlpha":1,
+        "cursorColor":"#258cbb",
+        "limitToGraph":"g1",
+        "valueLineAlpha":0.2
+    },
+    "categoryField": "date",
+    "categoryAxis": {
+        "parseDates": true,
+        "dashLength": 1,
+        "minorGridEnabled": true
+    },
+    "export": {
+        "enabled": true
+    },
+    "dataProvider": gon.chart_data
+  });
+
+//actions to the sidebar when resolving errors
+  $('#resolve-button').click(function(e){
+    var id, element;
+    id = $('.error-content').attr('data-id');
+    e.stopPropagation();
+    element = $(".errors-sidebar-elements[data-id='"+id+"']");
+    if(element){
+      $(element).fadeOut(500, function() { $(this).remove(); });
+    }
+  });
+
+  $('#multiple-resolve').click(function(){
+    var checked;
+    checked = $('.list-group input:checked');
+    if(checked.length > 0){
+      checked.each(function() {
+        $(this).closest('.errors-sidebar-elements').fadeOut(500, function() { $(this).closest('.errors-sidebar-elements').remove(); });
+      });
+    }
+    else{
+      alert("Please make a selection first!");
+    }
+  });
+};
+$(document).ready(ready);
+$(document).on('page:load', ready);
```

Comment:
nu inteleg de ce ai facut chestia asta. Metoda prin care trebuie sa scrii codul js ca sa mearga cu turbolinks e asta: https://github.com/kossnocorp/jquery.turbolinks#not-working-with-documentonready `$(function () { /* ... */ });`